rust-runtime/aws-smithy-runtime/src/client/interceptors.rs

```diff
@@ -65,6 +65,11 @@ macro_rules! interceptor_impl_fn {
             runtime_components: &RuntimeComponents,
             cfg: &mut ConfigBag,
         ) -> Result<(), InterceptorError> {
+            tracing::trace!(concat!(
+                "running `",
+                stringify!($interceptor),
+                "` interceptors"
+            ));
             let mut result: Result<(), (&str, BoxError)> = Ok(());
```

Comment:
if we logging here, may be nice to log how many too

Reply:
We don't actually know the count until we've looped over them since we only have an iterator here.